package.json

```diff
@@ -66,6 +66,8 @@
     "@babel/preset-env": "^7.12.17",
     "@babel/preset-react": "^7.12.13",
     "@fullhuman/postcss-purgecss": "^3.0.0",
+    "@material-ui/core": "^4.12.3",
```

Comment:
Is it possible to use version 5?

Reply:
The issue is the react version, mui v 5 it requires 17 and we're on 16.

Reply:
Following up from our slack message. Yes, please update to React version 17. This is in flight everywhere.